client/app/lib/components/ProviderWrapper.jsx

```diff
@@ -40,6 +40,11 @@ const themeSettings = {
         color: 'white !important',
       },
     },
+    MuiDialog: {
```

Comment:
Whats this for?

Reply:
Tooltips were being cut off in the MuiDateTimePicker. Seems to be a common occurrence with our custom components (e.g. DataTable pagination). This css fix resolves it.

![image](https://user-images.githubusercontent.com/9080974/177465707-3d704bff-5c2f-40ee-afca-41448eaac88a.png)